Cinema system/Cinema system/Controllers/CinemasController.cs

```diff
@@ -14,254 +12,171 @@ public class CinemasController : ControllerBase
 
         private readonly ICinemaService _сinemaService;
 
+        private const string ErrorOfCinemaNonexistence = "This cinema does not exists.";
+        private const string ErrorOfCinemaExistence = "This cinema is already exists.";
+        private const string ErrorOfHallNonexistence = "This hall does not exist in this cinema.";
+        private const string ErrorOfHallExistence = "This hall is already exists in this cinema.";
+
         public CinemasController(ICinemaService сinemaService)
         {
             _сinemaService = сinemaService;
         }
 
-        private IEnumerable<string> ErrorMessages(ModelStateDictionary model)
-        {
-            return model.Values
-                .SelectMany(x => x.Errors)
-                .Select(x => x.ErrorMessage);
-        }
-
         [HttpGet]
         public IActionResult GetCinemas()
         {
-            var response = new GetResponse<IEnumerable<CinemaView>>
-            {
-                IsSuccessful = true,
-                RequestedData = _сinemaService.GetCinemas()
-            };
-
-            return Ok(response);
+            return Ok(new GetResponse<IEnumerable<CinemaView>>(_сinemaService.GetCinemas()));
         }
 
         [HttpGet("{cinemaId}")]
         public IActionResult GetCinema(int cinemaId)
         {
-            var response = new GetResponse<CinemaView>();
-
             if (!_сinemaService.CheckCinemaExists(cinemaId))
             {
-                response.IsSuccessful = false;
-                response.ErrorDescriptions.Add("This cinema does not exists.");
-                return NotFound(response);
+                return NotFound(new GetResponse<CinemaView>(ErrorOfCinemaNonexistence));
```

Comment:
Why don't return base class?
Applies to all error responses in this controller.